code/missionui/missionweaponchoice.h

```diff
@@ -12,6 +12,8 @@
 #ifndef __MISSION_WEAPON_CHOICE_H__
 #define __MISSION_WEAPON_CHOICE_H__
 
+#include "mission/missionparse.h"
```

Comment:
I don't believe this include is actually needed here and could be replaced with a forward declaration of `struct team_date` since `wl_init_pool` only takes a pointer. This is however just a maintainability recommendation and shouldn't affect anything. Other than that this looks good.

Reply:
I tried a forward declaration originally within the header and couldn't get it to build without error. Perhaps I did it wrong.

Reply:
After some more inspection, it looks like `team_data` is an anonymous struct, hence why it can't be forward declared. If it was changed from:

```
typedef struct {
...
} team_data;
```
to:
```
typedef struct team_data {
...
} team_data;
```

It would work.

Reply:
I feel like that's outside the scope of this PR, but I'll add it if you want me to.

Reply:
It's fine. We can handle it after the Release if need be.